hal-x86_64/src/serial.rs

```diff
@@ -64,6 +64,7 @@ pub struct Nonblocking {
 }
 
 impl Port {
+    #[allow(clippy::unreadable_literal)]
```

Comment:
nit/TIOLI: this shoud probably just have an underscore stuck in it to placate clippy, rather than allowing the lint?

Reply:
Sure. I don't have strong opinions, but vaguely preferred the no-underscore version of the number, so stuck with it. I'll switch to adding the underscore.